src/rv32.tex

```diff
@@ -1213,7 +1213,24 @@ \section{Memory Ordering Instructions}
   predecessor} set preceding the FENCE.
 Chapter~\ref{ch:memorymodel} provides a precise description of the
 RISC-V memory consistency model.
-  
+
+The FENCE instruction also orders memory reads and writes made by the
+hart as observed by memory reads and writes made by an external
+device.  However, FENCE does not order observations of events made by
+an external device using any other signaling mechanism.
+
```

Comment:
I prefer this inclusive language. Contrasting memory vs other channels is excellent.